data/en/mbs_0203.json

```diff
@@ -151,6 +151,7 @@
                         "mandatory": true,
                         "q_code": "110",
                         "type": "Unit",
+                        "length": "short",
```

Comment:
The default is `short` do want to have to specify this on everything?
we need to be consistent, as for `use_plural_for_label` we are not specifying it everywhere but for `length` we are.

Reply:
This was to fix a bug that I bumped into.  Originally, because it was short by default, I wasn't defining this.  It then rendered the unit name instead of the unit because in the jinja template, when resolving answer.length, it was passing through None or '' (I didn't check), but it was overriding the default.

Is there a way to get around this?